core/templates/components/skill-selector/select-skill-modal.component.html

```diff
@@ -11,6 +11,7 @@ <h3>
                         [categorizedSkills]="categorizedSkills"
                         [untriagedSkillSummaries]="untriagedSkillSummaries"
                         [allowSkillsFromOtherTopics]="allowSkillsFromOtherTopics"
+                        [addedSkillsIds]="addedSkillsIds"
```

Comment:
addedSkillsIds --> skillIdsToExclude

(when naming here, you are looking at this from the perspective of the skill selector)

Reply:
done